guides/source/active_storage_overview.md

```diff
@@ -346,12 +349,12 @@ gcs: &gcs
 
 private_gcs:
   <<: *gcs
-  credentials: <%= Rails.root.join("path/to/private_keyfile.json") %>
+  credentials: <%= Rails.root.join("path/to/private_key.json") %>
   bucket: ""
 
 public_gcs:
   <<: *gcs
-  credentials: <%= Rails.root.join("path/to/public_keyfile.json") %>
+  credentials: <%= Rails.root.join("path/to/public_key.json") %>
```

Comment:
These aren't related to the PR body, but it wasn't worth another PR... this just prevents an ugly line break. Compare:

<img width="662" alt="image" src="https://user-images.githubusercontent.com/509837/211229319-3dbc8c7c-bbe8-4c22-b004-48c04fa03e4b.png">